### DISCUSSION TITLE
Has select-simd been integrated?

### DISCUSSION THREAD

**Discussion by Yifei-yang7** [minio/minio#11848] [Q&A] [answered]
Hi,

I'm trying to use S3 Select in MinIO and get a question about performance.

I referred to this [link](https://www.youtube.com/watch?v=eWbO3D7JM-4) where the demo shows using c5.2xlarge instance to query 3.5GB CSV file in 0.3 sec.
I also deployed MinIO in a single c5.2xlarge node and tried to query my 2GB CSV file, but it took me 13 sec. I run by:
`time ./mc sql --query "select count(*) from s3Object where lo_quantity <= 5" local/test/2GB_lineorder_0.csv`

Is there anything I did wrongly or missed in configuration before querying, that compared to the demo my performance is more than 20x worse? I also noticed this demo is from "select-simd-integration" branch, but seems not to appear in the current repo. Has this package been integrated? If not is there a simple way to make it available in MinIO Client just as the demo showed?
Thank you!

**Comment by klauspost** [comment, marked as answer]
For JSON, Yes, for CSV, no.
